tiapp.xml

```diff
@@ -38,7 +36,7 @@
     </iphone>
     <android xmlns:android="http://schemas.android.com/apk/res/android">
        <manifest>
-           <uses-sdk android:targetSdkVersion="21"/>
+           <uses-sdk android:targetSdkVersion="23"/>
```

Comment:
We can remove this. When the app is imported, this will be filled up automatically.

You can see it here that it is removed as well: https://github.com/appcelerator-developer-relations/Sample.RSS/commit/751a7fc22a0df85c6eca5bb0ed27aa8175bf9690